pages/studentPortal/web/React/props.tsx

```diff
@@ -48,47 +48,40 @@ export async function getServerSideProps({ params }) {
     },
     {
       component: "description",
-      text:
-        "Recall from the last lesson that React components are Javascript functions that return HTML. Props are simply the inputs or arguments to those functions",
+      text: "Recall from the last lesson that React components are Javascript functions that return HTML. Props are simply the inputs or arguments to those functions",
```

Comment:
I don't think you meant to touch all these text lines, but doesn't look like you changed anything... just a reminder to be careful here